supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,95 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+=== Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
```

Comment:
What's the alternative here? I don't see what this guideline means/is helping to say.